packages/rich-text/src/plugins/Hyperlink/HyperlinkModal.tsx

```diff
@@ -119,15 +119,17 @@ export function HyperlinkModal(props: HyperlinkModalProps) {
       <React.Fragment>
         <ModalContent>
           <Form>
-            <FormControl id="link-text" isRequired>
-              <FormControl.Label>Link text</FormControl.Label>
-              <TextInput
-                testId="link-text-input"
-                name="link-text"
-                value={linkText}
-                onChange={(event) => setLinkText(event.target.value)}
-              />
-            </FormControl>
+            {!props.linkType && (
```

Comment:
Are we sure that's the only way to detect "editing" mode?

Reply:
Not sure it's the only way but it's enough https://github.com/contentful/field-editors/blob/2f75688b640efb39506ba395041fd333098b8077/packages/rich-text/src/plugins/Hyperlink/HyperlinkModal.tsx#L239